backend/trpc/index.ts

```diff
@@ -378,7 +378,7 @@ export const appRouter = router({
       }
     }),
 
-  // Delete notebook
+  // Delete notebook (soft delete)
   deleteNotebook: authedProcedure
```

Comment:
Could probably call this `softDeleteNotebook` now, so that someday when we bring in real delete that has a way to go.